lib/membrane/time.ex

```diff
@@ -17,7 +17,7 @@ defmodule Membrane.Time do
             native_units: 1, native_unit: 0, nanoseconds: 1, nanosecond: 0, second: 0, seconds: 1}
 
   @typedoc """
-  The time represented in Membrane's native time units - nanoseconds.
+  The time represented in Membrane's native time units.
```

Comment:
actually the word 'native' usually relates to OS in this context, so I'd remove it as well :P
```suggestion
  The time represented in Membrane's time units.
```